Resources/views/Collector/db.html.twig

```diff
@@ -257,7 +257,8 @@
         }
 
         function expandAllQueries(button) {
-            var queries = document.getElementsByClassName('query-section'),
+            var conn = button.getAttribute('data-connection'),
+                queries = document.getElementById('queries-' + conn).getElementsByClassName('query-section'),
```

Comment:
Before my change, it was also `getElementsByClassName()`, and there are a few more calls. That's why I didn't bother to verify the compatibility.

Thanks, I'll rewrite it  using the best practices ;)